js/hw8.js

```diff
@@ -0,0 +1,8 @@
+const glueTogetherWords = function () {
```

Comment:
read the task attentively. Your function have to accepts 2 strings and then concat them/ not local variables